packages/block-editor/src/components/block-preview/index.js

```diff
@@ -128,6 +161,10 @@ export function useBlockPreview( { blocks, props = {}, layout } ) {
 			value={ renderedBlocks }
 			settings={ settings }
 		>
+			<SyncStyleOverridesWithParent
```

Comment:
Do we really need to sync here, can't we just use `EditorStyles` component to render these overrides here?

Reply:
I think syncing is wrong because if the "preview" is unmounted, why should its overrides linger. They are just the styles for the preview not for any parent editor.

Reply:
> can't we just use EditorStyles component to render these overrides here?

Oh, good question, I'll try that out!